### PR TITLE
Add RockyLinux 9.0

### PR DESCRIPTION
**Pull request by phillipross**
Signed-off-by: Phillip Ross <phillip.w.g.ross@gmail.com>

## Description
Add RockyLinux 9.0 box

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: packer_templates/rockylinux/http/9/ks.cfg [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+             BROWSER_ONLY=no
+             BOOTPROTO=dhcp
+             DEFROUTE=yes
+IPV4_FAILURE_FATAL=no
+IPV6INIT=yes
+IPV6_AUTOCONF=yes
+IPV6_DEFROUTE=yes
+             IPV6_FAILURE_FATAL=no
+             IPV6_ADDR_GEN_MODE=stable-privacy
+             NAME=eth0
```
Comment (Stromweld) [inline packer_templates/rockylinux/http/9/ks.cfg:72]: Style:
please remove indents

**Review by Stromweld — CHANGES_REQUESTED** [review]
Looks good just a couple of copy paste styling issues to fix.

File: packer_templates/rockylinux/rockylinux-9.0-x86_64.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    "git_revision": "__unknown_git_revision__",
+    "guest_additions_url": "",
+    "headless": "",
+    "http_directory": "{{template_dir}}/http",
+    "http_proxy": "{{env `http_proxy`}}",
+    "https_proxy": "{{env `https_proxy`}}",
+    "hyperv_generation": "1",
+    "hyperv_switch": "bento",
+    "iso_checksum": "1d48e0af63d07ff4e582a1819348e714c694e7fd33207f48879c2bc806960786",
+    "iso_name": "Rocky-9.0-x86_64-dvd1.iso",
```
Comment (Stromweld) [inline packer_templates/rockylinux/rockylinux-9.0-x86_64.json:184]: iso_name should be "Rocky-9.0-x86_64-dvd.iso"

File: packer_templates/rockylinux/rockylinux-9.0-x86_64.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    "ks_path": "9/ks.cfg",
+    "memory": "1024",
+    "mirror": "http://download.rockylinux.org/pub/rocky",
+    "mirror_directory": "9.0/isos/x86_64",
```
Comment (Stromweld) [inline packer_templates/rockylinux/rockylinux-9.0-x86_64.json:188]: mirror_directory should be 9/isos/x86_64

**Review by Stromweld — CHANGES_REQUESTED** [review]
I was getting errors trying to download image untill these changes were made to my local copy.

File: packer_templates/rockylinux/rockylinux-9.0-x86_64.json [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    "disk_size": "65536",
+    "git_revision": "__unknown_git_revision__",
+    "guest_additions_url": "",
+    "headless": "",
+    "http_directory": "{{template_dir}}/http",
+    "http_proxy": "{{env `http_proxy`}}",
+    "https_proxy": "{{env `https_proxy`}}",
+    "hyperv_generation": "1",
+    "hyperv_switch": "bento",
+    "iso_checksum": "1d48e0af63d07ff4e582a1819348e714c694e7fd33207f48879c2bc806960786",
```
Comment (Stromweld) [inline packer_templates/rockylinux/rockylinux-9.0-x86_64.json:183]: iso_checksum also needs to be "55dbf904b35969be9f6d96f042b7470d1a3cb5c953bba47ab8f79740560d8a8d"

File: packer_templates/rockylinux/rockylinux-9.0-x86_64.json (hunk 2 above)
Comment (phillipross) [reply packer_templates/rockylinux/rockylinux-9.0-x86_64.json:184]: Very odd... you're absolutely correct about the iso_name and checksum.  It actually does build for me, but I wonder if it's working off of a packer cache or something.  I'll try to clear out cache and see if I can make it fail, and then fix the paths 😁

Thanks for the review and sanity check, I'll push the changes when I figure out why it's working when it shouldn't and resolve everything.

**Review by Stromweld — APPROVED** [review]
testing on my local workstation passed.

**Pull request merged**